nengo/utils/builder.py

```diff
@@ -177,8 +177,8 @@ def _create_replacement_connection(c_in, c_out):
     return c
 
 
-def remove_passthrough_nodes(objs, connections,  # noqa: C901
-        create_connection_fn=_create_replacement_connection):
```

Comment:
I'd assume that if you do a line break in the argument list, all lines have to start with the same indentation. That is either
```python
def remove_passthrough_nodes(objs, connections,
                             create_connection_fn=_create_replacement_connection):
```
or how it is done now.

But I'm not 100% sure.